src/Contexts/AuthDialogContext/useAuthDialog.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+export const useAuthDialog = () => {
+  const [authType,setAuthType] = useState(null);
+  const [creds,setCreds] = useState({});
+
+  const handleAuthCreds = (e) => {
```

Comment:
If not using arrow function in the code. If possible remove arrow function.

Reply:
removed this file .